src/main/java/com/google/sps/utils/QuizTimingPropertiesUtils.java

```diff
@@ -52,9 +52,9 @@
         "Which plant would look the best outside in a garden?",
         "Which plant would you give as a gift?"
     ));
-
+ 
     //This function gets the the "quiz_timestamp" property of the entity that is fed into the function
-    public Long getTimestampProperty(String entity, DatastoreService datastore) {
+    public Long getQuizTimestampProperty(String entity, String idProperty, String idPropertyValue, DatastoreService datastore) {
```

Comment:
idProperty and idPropertyValue are ambiguous. What do they actually represent? the variable names should be altered to reflect that.